### ISSUE TITLE
fix: Accessibility Issue in FluentDialogHeader

### ISSUE THREAD

**Issue by tlmii** [microsoft/fluentui-blazor#848]
# 🐛 Bug Report

The dismiss button in the header of the dialog doesn't have a title/aria-label

## 💻 Repro or Code Sample

This can be seen on https://www.fluentui-blazor.net/Dialog: 
![image](https://github.com/microsoft/fluentui-blazor/assets/9613109/8806eb57-6229-4dc3-857b-c05137156cb9)

**Comment by dvoituron** [comment]
You're right. But my problem is more with the `FluentButton` (used to display the Dismiss icon). I'm fixing it for the next version.

**Comment by dvoituron** [comment]
Fixed in PR #853
